client/src/components/reader/ReturnMenuBody.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import HigherOrder from 'containers/global/HigherOrder';
+
+export default class ReturnMenuBody extends Component {
+
+  static propTypes = {
+    projectId: PropTypes.string,
+    projectTitle: PropTypes.string,
+    toggleSignInUpOverlay: PropTypes.func,
+    moreLink: PropTypes.string,
+  };
```

Comment:
Because there's no logic in the render method to handle the absence of these props, it seems to me that they should all be flagged as required for this component.